cmd/osde2ectl/get/cmd.go

```diff
@@ -99,28 +132,67 @@ func run(cmd *cobra.Command, argv []string) error {
 	if properties := cluster.Properties(); properties["MadeByOSDe2e"] != "true" {
 		return fmt.Errorf("Cluster was not created by osde2e")
 	}
-	log.Printf("Cluster name - %s and Cluster ID - %s", cluster.ID(), cluster.Name())
+	log.Printf("Cluster name - %s and Cluster ID - %s", cluster.Name(), cluster.ID())
 
 	if kubeconfigStatus {
-		err := setKubeconfig(clusterID)
+		content, err := setKubeconfig(clusterID)
 		if err != nil {
 			return fmt.Errorf("Error getting the cluster's kubeconfig - %s", err)
 		}
+
+		filename := cluster.Name() + "-kubeconfig.txt"
+
+		var filePath string
+		if viper.GetString(config.Kubeconfig.Path) != "" {
+			_, err := os.Stat(viper.GetString(config.Kubeconfig.Path))
```

Comment:
I think we should be more explicit here. This assumes that any error trying to stat the path will be because the directory doesn't exist. Check using https://golang.org/pkg/os/#IsNotExist for this.

Also, let's use the args struct directly instead of viper here.